js/filter.js

```diff
@@ -1,27 +1,94 @@
 'use strict';
+//import L from 'leaflet';
+/* global _:readonly */
 
-import {removeMarkers} from './map.js';
-import {addOtherPins} from './map.js';
+import {
+  removeMarkers
+} from './map.js';
+import {
+  addOtherPins
+} from './map.js';
```

Comment:
Надо объединять импорты из одного модуля
```suggestion
import { removeMarkers, addOtherPins } from './map.js';
```